MaterialComponents.podspec

```diff
@@ -459,7 +459,7 @@ Pod::Spec.new do |s|
       sss.dependency "MaterialComponents/Typography"
 
       sss.dependency "MaterialComponents/private/Math"
-      sss.dependency "MaterialComponents/private/RTL"
+      sss.dependency "MDFInternationalization"
```

Comment:
Since this component does not depend on MDCRTL anymore, you should also remove the rest of `MatiralRTL.h` imports in:
* `MDCMultilineTextField.m`
* `MDCTextInputControllerFilled.m`
* `MDCTextInputControllerFullWidth.m`
* `MDCTextInputControllerLegacyDefault.m`
* `MDCTextInputControllerLegacyFullWidth.m`

Reply:
I think yours covers that. We’ll merge it first.